queries/defillama/v1/tvl.sql

```diff
@@ -16,7 +16,7 @@ FROM
   JOIN omnipool_asset oa ON leb.height = oa.block
   JOIN token_metadata tm ON oa.asset_id = tm.id
 WHERE CASE
-      WHEN :asset::text IS NOT NULL
```

Comment:
@green-jay is this change intentional?

Reply:
yes but it can stay either way, this was just part of changing asset to integer everywhere more explicitely

Reply:
are you sure? According to the defillama endpoint specification, it passes ticker

Reply:
ah ofc my bad, that should be reverted

Reply:
Defillama TVL endpoint should be removed altogether, they are not using it. They are using only /volume endpoint